### ISSUE TITLE
Error while deleting version with plus sign in it

### ISSUE THREAD

**Issue by MathMagique** [blue-yonder/devpi-cleaner#3]
I encountered the following error when performing the deletion for some packages which were marked for deletion as expected (displaying works):

```
Traceback (most recent call last):
  File "/tmp/devpi-cleaner/venv/bin/devpi-cleaner", line 11, in <module>
    sys.exit(main())
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/devpi_cleaner/cli.py", line 45, in main
    remove_packages(client, packages, args.force)
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/devpi_cleaner/client.py", line 61, in remove_packages
    client.remove('{name}=={version}'.format(name=package.name, version=package.version))
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/devpi_plumber/client.py", line 129, in remove
    return self._execute('remove', '-y', *args)
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/devpi_plumber/client.py", line 59, in _execute
    devpi(args)
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/devpi/main.py", line 30, in main
    return method(hub, hub.args)
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/devpi/list_remove.py", line 118, in main_remove
    req = parse_requirement(args.spec)
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/devpi_common/metadata.py", line 152, in parse_requirement
    return Requirement.parse(s)
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/pkg_resources.py", line 2612, in parse
    reqs = list(parse_requirements(s))
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/pkg_resources.py", line 2550, in parse_requirements
    line, p, specs = scan_list(VERSION,LINE_END,line,p,(1,2),"version spec")
  File "/tmp/devpi-cleaner/venv/local/lib/python2.7/site-packages/pkg_resources.py", line 2528, in scan_list
    "Expected ',' or end-of-list in",line,"at",line[p:]
ValueError: ("Expected ',' or end-of-list in", 'package==0.6b6.post0.dev66+g9a4b6a4', 'at', '+g9a4b6a4')

```

**Comment by StephanErb** [comment]
Which setuptools & pip version are you using?

**Comment by MathMagique** [comment]
I use pip version 1.5 and setuptools 2.0.2.

```
> pip list
argparse (1.2.1)
check-manifest (0.25)
devpi-cleaner (0.1.0)
devpi-client (2.3.0)
devpi-common (2.0.6)
devpi-plumber (0.2.5)
devpi-server (2.2.2)
execnet (1.3.0)
itsdangerous (0.24)
PasteDeploy (1.5.2)
pip (1.5)
pkginfo (1.2.1)
pluggy (0.3.0)
py (1.4.30)
pyramid (1.5.7)
PyYAML (3.11)
repoze.lru (0.6)
requests (2.7.0)
setuptools (2.0.2)
six (1.9.0)
tox (2.1.1)
translationstring (1.3)
twitter.common.contextutil (0.3.3)
twitter.common.dirutil (0.3.3)
twitter.common.lang (0.3.3)
venusian (1.0)
virtualenv (13.1.0)
waitress (0.8.9)
WebOb (1.4.1)
wsgiref (0.1.2)
zope.deprecation (4.1.2)
zope.interface (4.1.2)
```

**Comment by MathMagique** [comment]
Works after updating setuptools to current version (18.0.1)

**Comment by StephanErb** [comment]
Fixed in commit 80eb6a64
